### DISCUSSION TITLE
i am having hard time with my developers.

### DISCUSSION THREAD

**Discussion by keshavgupta848101** [thebergamo/react-native-fbsdk-next#197] [Q&A]
Hello everyone,
I am Keshav me and my friend we hire two developers to integrate Facebook SDK to one of our apps and now developers are saying that they have made the app but to get it verified/get authorization from facebook it will take Facebook around two months and this is making our work pace slow. Can anyone from the community answer this question how much time it will take to get verified/get authorization from Facebook?

**Comment by mikehardy** [comment]
Last time I did the process it took just a few business days. But I don't think anyone here can say for sure. You can always ask facebook directly, using your account there, that's the way to get real answers that are current I think
